src/citra_qt/configuration/configure_cheats.h

```diff
@@ -20,7 +25,7 @@ class ConfigureCheats : public QWidget {
     Q_OBJECT
 
 public:
-    explicit ConfigureCheats(u64 title_id_, QWidget* parent = nullptr);
+    explicit ConfigureCheats(Core::System& system, u64 title_id, QWidget* parent = nullptr);
```

Comment:
```suggestion
    explicit ConfigureCheats(Core::System& system, u64 title_id_, QWidget* parent = nullptr);
```
to be consistent with the cpp file

Reply:
The dash is added to avoid variable shadowing with the class member. The header should have the proper parameter name.